benchmark/dgram/bind-params.js

```diff
@@ -0,0 +1,42 @@
+﻿'use strict';
+
+const common = require('../common.js');
+const dgram = require('dgram');
+
+const numberOfOperations = 1e4;
+
+const configs = {
+  n: [numberOfOperations],
+  port: ['true', 'false'],
+  address: ['true', 'false'],
+};
+
+const bench = common.createBenchmark(main, configs);
+
+function main(conf) {
+  const n = +conf.n;
+  const port = conf.port === 'true' ? 0 : undefined;
+  const address = conf.address === 'true' ? '0.0.0.0' : undefined;
+
+  if (port !== undefined && address !== undefined) {
+    bench.start();
+    for (let i = 0; i < n; i++) {
+      dgram.createSocket('udp4').bind(port, address).unref();
+    }
+    bench.end(n);
+  } else if (address !== undefined) {
+    // Skip: the same as the previous case.
```

Comment:
That's basically what I was suggesting, yes. Except perhaps it would be better to reverse the checks to match the same order used in the first conditional:

    } else if (port !== undefined && address === undefined) {

or we could just remove the middle section entirely and it should still work as-is:

    } else if (port !== undefined) {